report-api/src/test/java/no/sikt/nva/data/report/api/fetch/formatter/ExpectedCsvFormatter.java

```diff
@@ -0,0 +1,83 @@
+package no.sikt.nva.data.report.api.fetch.formatter;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ExpectedCsvFormatter {
```

Comment:
This method formats CSV input to the format returned by the Jena formatter — the code is deceptively simple.